cobalt/user_agent/BUILD.gn

```diff
@@ -24,4 +24,12 @@ static_library("user_agent") {
     "//starboard/common",
     "//v8:v8_version",
   ]
+
+  if (is_android) {
+    import("//build/config/android/abi.gni")
+    defines = [
+      # Enable compile-time decisions based on the ABI
+      "ANDROID_ABI=$android_app_abi",
```

Comment:
I also see this used in https://github.com/youtube/cobalt/blob/main/starboard/android/shared/system_get_property.cc#L39, is that file no longer used?

Reply:
yes, that function is no longer used, this file can be cleaned up or fully migrated in the future.